.travis.yml

```diff
@@ -4,12 +4,17 @@ group: edge
 
 jdk: oraclejdk8
 
+env:
+  global:
+    # DockerHub password for aschen (amaret@kuzzle.io) account
```

Comment:
not sure you want to make that kind of information public

Reply:
The password is encrypted